telemetry/setup/setup.go

```diff
@@ -4,26 +4,57 @@ import (
 	"os"
 
 	"github.com/goravel/framework/packages"
+	"github.com/goravel/framework/packages/match"
 	"github.com/goravel/framework/packages/modify"
 	"github.com/goravel/framework/support/path"
 )
 
 func main() {
+	const (
+		fileTelemetryConfig = "telemetry.go"
+		fileLoggingConfig   = "logging.go"
+		providerTelemetry   = "&telemetry.ServiceProvider{}"
+		importLog           = "github.com/goravel/framework/telemetry/instrumentation/log"
+		aliasLog            = "telemetrylog"
+		keyLoggingChannels  = "logging.channels"
+		keyOtel             = "otel"
+		configSnippetOtel   = `map[string]any{
+		"driver": "custom",
+		"via":    telemetrylog.NewTelemetryChannel(),
+	}`
+	)
+
 	setup := packages.Setup(os.Args)
 	stubs := Stubs{}
-	telemetryConfigPath := path.Config("telemetry.go")
-	telemetryFacadePath := path.Facades("telemetry.go")
-	telemetryServiceProvider := "&telemetry.ServiceProvider{}"
-	modulePath := setup.Paths().Module().Import()
-	facadesPackage := setup.Paths().Facades().Package()
+	paths := setup.Paths()
+
+	pathConfigTelemetry := path.Config(fileTelemetryConfig)
+	pathFacadesTelemetry := path.Facades(fileTelemetryConfig)
+	pathConfigLogging := path.Config(fileLoggingConfig)
+
+	importModule := paths.Module().Import()
+	packageFacades := paths.Facades().Package()
+
```

Comment:
They are noun, so the new names are better.

```suggestion
	moduleImport := paths.Module().Import()
	facadesPackage := paths.Facades().Package()

```